require_file.php

```diff
@@ -1,7 +1,11 @@
 <?php
-/**
- * Created by PhpStorm.
- * User: academy
- * Date: 02/10/2018
- * Time: 10:16
- */
\ No newline at end of file
+
+$db = new PDO('mysql:host=127.0.0.1;dbname=portfoliodb','root');
+
+$aboutme_text = $_POST['text'];
+
+$query = $db->prepare("UPDATE `about_me` SET `text`=:text WHERE `id`=1");
```

Comment:
try to put these into functions.
as this is procedural code at the moment.

Reply:
speaking of functions do you have any tests files?

Reply:
quick reminder you don't have to test functions that touch the database. so maybe try to separate the functions and where you call to them. ask me to elaborate on this.